packages/opentelemetry/lib/index.js

```diff
@@ -21,6 +26,7 @@ const logger = require('@dotcom-reliability-kit/logger');
 
 const USER_AGENT = `FTSystem/${appInfo.systemCode} (${packageJson.name}/${packageJson.version})`;
 const TRACING_USER_AGENT = `${USER_AGENT} (${traceExporterPackageJson.name}/${traceExporterPackageJson.version})`;
+const METRIC_USER_AGENT = `${USER_AGENT} (${matricExporterPackageJson.name}/${matricExporterPackageJson.version})`;
```

Comment:
Is the thinking here to distinguish this package from the default user-agent set by the OTel SDK?

If so it might be worth being aware of resource attributes, there's a few set by the SDK:

https://opentelemetry.io/docs/specs/semconv/resource/#telemetry-sdk

Maybe you want to utilise them, or something similar to keep track of what's in use? Very keen to discuss this if you want to do something different to the defaults, so we can get everyone on the same plan.